### ISSUE TITLE
The package.json missed storyboard

### ISSUE THREAD

**Issue by carlleton** [guigrpa/docx-templates#85]
may the package.json missing this libs:
1. storyboard
2. storyboard-listener-console
3. lodash

**Comment by vdechef** [comment]
Hi think those dependencies are for development only. Maybe we could add them in devDependencies part ?

**Comment by carlleton** [comment]
Hi, I used the docx-templates in a Vue project, maybe the development reason

**Comment by vdechef** [comment]
I also use docx-templates in a Vue project. I just checked the package.json of my project, and in the devDependencies part I have storyboard but not lodash. Still it works fine when running in dev mode or when building for production.
```
"devDependencies": {
    ...
    "storyboard": "^3.1.4",
    "storyboard-listener-console": "^3.1.4",
    ...
  },
```

**Comment by carlleton** [comment]
You are right, lodash is just what I need

**Comment by nmummau** [comment]
I'm also trying to use this in a Vue project, at this point without success.
When running my project I get:
```
ERROR  Failed to compile with 2 errors
These dependencies were not found:
* storyboard in ./node_modules/docx-templates/lib/debug.js
* storyboard-listener-console in ./node_modules/docx-templates/lib/debug.js
```

**Comment by jjhbw** [comment]
Should be fixed with the 4.0.0 release. Thanks for reporting!
